charts/images.yaml

```diff
@@ -161,8 +161,8 @@ images:
   tag: "1.22.5"
 - name: node-problem-detector
   sourceRepository: github.com/kubernetes/node-problem-detector
-  repository: k8s.gcr.io/node-problem-detector/node-problem-detector
-  tag: "v0.8.7"
+  repository: eu.gcr.io/gardener-project/3rd/node-problem-detector
+  tag: "v0.8.10"
```

Comment:
`v0.8.10` is ambiguous: the image you are referencing is built from a different revision than the tag `v0.8.10`.

Please
- add a new commit, that bumps `VERSION` to `v0.8.10-gardener.1`
- tag this commit as `v0.8.10-gardener.1`
- let CI build a new image from this git tag and with a corresponding image tag